### PR TITLE
Component helper instead of block slots

### PR DESCRIPTION
**Pull request by srowhani**
#major#

### CODE REVIEW

File: addon/components/nav-category.js [hunk 1]
```diff
@@ -9,36 +10,40 @@ const {
...
-      navService.dismiss()
-      later(() => {
-        navService.set('_activeCategory', name)
-      }, 100)
     } else {
-      navService.set('_activeCategory', name)
+      if (typeof activeCategory === 'string') {
+        navService.dismiss()
+        later(() => {
+          navService.set('_activeCategory', name)
```
Comment (juwara0) [inline addon/components/nav-category.js]: You can destructure the `set` as well.

`set(navService, '_activeCategory', name)` 

Remember to define it at the top.

File: addon/components/nav-category.js [hunk 2]
```diff
@@ -9,36 +10,40 @@ const {
...
+        }, 200)
+      } else {
+        navService.set('_activeCategory', name)
```
Comment (juwara0) [inline addon/components/nav-category.js]: same note about destructing the `set`.

**Comment by sglanzer** [comment]
Is this PR relevant anymore?  I saw you merged the other PR

**Comment by srowhani** [comment]
@sglanzer I accidentally pushed to master on ciena-frost after updating the `gh-pages` branch. I reverted the changes and reopened the PR

**Comment by sglanzer** [comment]
Aaah, ok - I made my comments on the other one, should still apply :)

**Comment by sglanzer** [comment]
Approved - looks good

File: package.json [hunk 3]
```diff
@@ -32,21 +31,21 @@
...
-    "ember-cli-mirage": "~0.1.11",
     "ember-cli-mocha": "^0.10.1",
-    "ember-cli-notifications": "^3.2.0",
+    "ember-cli-notifications": "^4.0.6",
     "ember-cli-uglify": "^1.2.0",
+    "ember-code-snippet": "1.4.0",
     "ember-computed-decorators": "0.2.2",
     "ember-concurrency": "0.7.8",
-    "ember-data": "^2.4.0",
+    "ember-data": "^2.8.0",
```
Comment (rox163) [inline package.json]: Remove ember data since I don't think it's used.  Otherwise keep in sync with ember and ember cli versions

**Comment by rox163** [comment]
:+1:

**Comment by rox163** [comment]
Looks like you have a legit test failure with this merge - https://travis-ci.org/ciena-frost/ember-frost-navigation/jobs/159984231#L2429
Looks like a liquid fire issue. Usually a bump to 0.25.0 fixes that.

**Pull request merged**